pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,117 @@
+# Test ownership
+Test ownership is critical to adopting a healthy testing culture at your organization. By defining test owners, your teams will become accountable for maintaining a fast and reliable test suite, ensuring confidence when you deploy your code.
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/team-management/permissions).
```

Comment:
Test Analytics has its own 'Access controls' docs page for this, with relevant links through to the general platform-level docs about teams, currently in the Pipelines section of the Buildkite Docs, but which should eventually be pulled out into its own 'Platform' docs section.

It'd be more sensible for this link to lead to the Test Analyics page equivalent first, since it also talks about test suite-level permissions.

```suggestion
Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/test-analytics/permissions).
```